lang/python/wiredtiger.i

```diff
@@ -927,6 +951,17 @@ typedef int int_void;
 		else:
 			return unpack(self.value_format, self._get_value())
 
+	def get_raw_key_and_value(self):
```

Comment:
Is there a reason you called this `get_raw_key_and_value` where the C API is `get_raw_key_value`? i.e: why did you choose to add the `and` in?

Reply:
Without the `and` something very odd is happening with the SWIG generated code, as the test will fail with a weird error. I'm investigating the underlying cause...